custom_components/bambu_lab/camera.py

```diff
@@ -67,7 +67,12 @@ async def stream_source(self) -> str | None:
             # rtsps://192.168.1.1/streaming/live/1
 
             parsed_url = urlparse(self.coordinator.get_model().camera.rtsp_url)
-            url = fr"{parsed_url.scheme}://bblp:{self._access_code}@{parsed_url.netloc}{parsed_url.path}"
+            if self.coordinator.get_model().info.mqtt_mode == "local":
+                # For unknown reasons the returned rtsp URL sometimes has a completely incorrect IP address in it for the host.
+                # Since we know the correct IP (but only in local mqtt connection mode), rewrite the URL to have that.
+                url = fr"{parsed_url.scheme}://bblp:{self._access_code}@{self._host}/{parsed_url.path}"
```

Comment:
Please remove the backslash before {parsed_url.path} as that variable is already prefixed with a backslash. Fixed line should be:
`url = fr"{parsed_url.scheme}://bblp:{self._access_code}@{self._host}{parsed_url.path}"`

Reply:
#306

Reply:
Fixed v1.9.3-dev2 release.

Reply:
In my testing though that doesn't break the camera entity - the error above seems likely to just be bugs in the firmware.